pyproject.toml

```diff
@@ -42,9 +42,8 @@ classifiers = [
 
 [project.optional-dependencies]
 cli = ["click>=8.1.7", "pyogrio>=0.8", "shapely>=2"]
-geopandas = ["geopandas>=0.13", "pandas>=2", "shapely>=2"]
+geopandas = ["geopandas>=0.13", "mapclassify>=2.8.1", "pandas>=2", "shapely>=2"]
```

Comment:
I think we can take out the dependency altogether from here. If someone hits the error they'll be instructed to install it.